packages/babel-plugin-transform-block-scoping/test/fixtures/general/issue-8128-for-of-after/output.js

```diff
@@ -1,9 +1,8 @@
-var _loop = function (_result) {
-  _result = otherValue;
+var _loop = function (result) {
+  result = otherValue;
   fn(() => {
-    _result;
+    result;
   });
-  result = _result;
```

Comment:
This is removed (and it was unnecessary) because now the loop block-scoped variable is transformed before transforming `for-of`.